### ISSUE TITLE
Why prepruning distillation?

### ISSUE THREAD

**Issue by mpiorczynski** [princeton-nlp/CoFiPruning#56]
Hi, I have a question about the intuition behind the prepruning distillation step. Why are you not initializing the student model from the teacher weights, instead of initializing it from scratch (/pretrained on MLM BERT checkpoint)?

**Comment by xiamengzhou** [comment]
Yes, I think initializing the teacher weights is completely fine and with the limited exploration we did, empirically they have very similar results :)
